lib/path/default.nix

```
@@ -94,6 +95,47 @@ let
 
 in /* No rec! Add dependencies on this file at the top. */ {
 
+  /* Append a subpath string to an absolute path value, the result is an absolute path value.
+
+    Like `<path> + ("/" + <string>)` but safer, because it errors for potentially unexpected results.
```

```suggestion
    Like `<<path>> + ("/" + <<string>>)` but safer, because it errors for potentially unexpected results.
```

`<path>` is syntax for `NIX_PATH`.

---

I changed it to just `path + ("/" + string)` now, I think that's nicer